### PR TITLE
ENDOC-467 Update the Google GKE install guide for 7.0

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**

### CODE REVIEW

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 1]
```diff
@@ -2,174 +2,149 @@
...
-These are the minimal instructions to prepare NGINX ingress using the Google Cloud Shell. To install it
-using your local `kubectl` or to vary other settings please see the more detailed documents here:
-<https://cloud.google.com/community/tutorials/nginx-ingress-gke> and <https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke>.
+6. Click the `Configure` button for the `GKE Standard` option. Unless otherwise indicated, you can keep the default configuration options.
 
-1. Initialize your user as a cluster-admin:
-```
+7. In the left menu, select `default-pool` → `Nodes`
+
+8. Select "e2-standard-2" as the Machine Type if you are setting up a basic test cluster for a single Entando Application. Additional CPU and memory may be required for a shared cluster containing multiple Entando Applications or to improve performance. Refer to [Appendix A](#appendix-a-configuring-clustered-storage) for details on clustered storage.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:26]: should Machine Type be backticked?

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 2]
```diff
@@ -2,174 +2,149 @@
...
+11. Click `Run in Cloud Shell`. Alternatively, connect your local `kubectl` to the GKE cluster.
+
+12. Run `kubectl get node` to verify your connection. The output should list the nodes in your cluster.
+
+### Install the NGINX Ingress Controller
+
+The following steps install the NGINX ingress controller to manage the ingresses for Entando services deployed by the operator. This is a simpler and more adaptable configuration for most users and environments. Users who require the GKE ingress controller (this is rare) can follow
+[the integration instructions provided by GKE](https://cloud.google.com/kubernetes-engine/docs/concepts/ingress) and then customize the service definition created by the Entando operator.
+
+These are the mimimum instructions to prepare the NGINX ingress using the Google Cloud Shell. Refer to <https://cloud.google.com/community/tutorials/nginx-ingress-gke> and <https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke> for installation using your local `kubectl` or to vary other settings.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:41]: second sentence, change links to text? easier to absorb >> For installation using your local `kubectl` or to vary other settings, refer to [Ingress with NGINX](https://cloud.google.com/community/tutorials/nginx-ingress-gke) and [GCE-GKE details](https://kubernetes.github.io/ingress-nginx/deploy/#gce-gke)

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 3]
```diff
@@ -2,174 +2,149 @@
...
 
 ```
 kubectl -n {NAMESPACE} patch ingress quickstart-eci-ingress --type='json' -p='[{"op": "replace", "path": "/spec/rules/0/http/paths/0/path", "value":"/k8s/"}]'
 ingress.extensions/quickstart-eci-ingress patched
 ```
 
-## Workaround for invalid login form, redirect when using TLS
+### Workaround
+
+Bypass an invalid login form and redirect when using TLS using this workaround.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:145]: a bit confusing >> Bypass an invalid login form and redirect when using TLS with this workaround 

not sure if this is what was meant

File: vuepress/docs/next/tutorials/getting-started/gke-install.md [hunk 4]
```diff
@@ -182,52 +157,50 @@ spec:
...
 
 ::: tip
-You can also scale an Entando Application without clustered storage using a `ReadWriteOnce (RWO)` policy by ensuring that the
-instances are all scheduled to the same node. This can be accomplished using taints on other nodes. Be aware of the pros and cons of scheduling
-instances to the same node. This will give you protection if the application instance itself dies or becomes unreachable and will help
-you get the most utilization of node resources. However, if the node dies or is shutdown you will have to wait for Kubernetes to reschedule the pods to a different node and your application will be down.
+You do not need clustered storage to scale an Entando Application if you schedule all instances to the same node using a `ReadWriteOnce (RWO)` policy and taints on other nodes. Be aware of the impact to node resource allocation, as well as recovery if your application fails or becomes unreachable. Note that if the node fais or is shutdown, your application will be unresponsive while Kubernetes reschedules the pods to a different node.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/getting-started/gke-install.md:170]: using a `ReadWriteOnce (RWO)` policy, and taints on other nodes. 

You only need that comma if the node using RWO and taints are different.  It makes the distinction more clear.

**Review by jyunmitch — APPROVED** [review]

**Review by nshaw — CHANGES_REQUESTED** [review]

**Review by nshaw — APPROVED** [review]

**Pull request merged**